File: src/test/java/twosum/SolutionTest.java
package twosum;

import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

class SolutionTest {

    private final Solution solution = new Solution();

    @Test
    public void testOne() {
        int[] nums = {2, 7, 11, 15};
        int target = 9;

        int[] indexes = solution.twoSum(nums, target);

        assertThat(indexes).containsExactly(0, 1);
    }

}

File: src/test/kotlin/dev/sanket/plusone/SolutionTest.kt
package dev.sanket.plusone

import assertk.assertThat
import assertk.assertions.containsExactly
import dev.sanket.dev.sanket.plusone.Solution
import org.junit.jupiter.api.Test

class SolutionTest {

    private val solution = Solution()

    @Test
    fun testOne() {
        val result = solution.plusOne(intArrayOf(1, 2, 3))

        assertThat(result).containsExactly(*intArrayOf(1, 2, 4))
    }
}